funcx_web_service/task_storage.py

```diff
@@ -0,0 +1,68 @@
+"""
```

Comment:
Any reason this shouldn't be part of funcX common? Doesn't seem there is anything specific to WebService here...

Reply:
Part of this integrates the TaskStorage with Flask. So that is application-specific.

I would have much preferred for `funcx-common` to implement the environment variable loading and failover in the absence of the S3 bucket. That was my original intent, when I planned to write it myself.

Reply:
Having flask specific stuff here makes sense to me.  Though the fail over logic definitely feels like that is something that *should* be in funcx-common.  Is there a reason we need that here besides removing the dependency on helm chart changes before we merge?  If there is, this is probably fine for expediency.  But eventually, we should  have the config parsing and "factory" logic for creating storages in funcx-common so all services can use the same logic and be configured in the same way.

Reply:
It's so easy to _not_ break main in this case, it's easily worth the cost. Not requiring S3 also strikes me as desirable until we have a plan for how we're going to support local development.

I'd love to do more work in funcx-common to support this better, but not until my PR there to restore proper test coverage is merged. I've piled up multiple PRs in other funcx repos, and it burns time to continually rebase them to keep them in a merge-able state.